src/classes/STG_InstallScript.cls

```diff
@@ -222,6 +222,22 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
+     /*******************************************************************************************************
+    * @description Disables Preferred Phone handling when upgrading to 1.79
+    * @return void
+    */
+    
+    global static void disablePreferredPhoneEnforcement(Version previousVersion) {
+        if(previousVersion != null && previousVersion.compareTo(new Version(1,79)) < 0) {
+            UTIL_Debug.debug('***Upgrading to version 1.79');
+            Hierarchy_Settings__c orgSettings = Hierarchy_Settings__c.getOrgDefaults();
```

Comment:
@skosgi1 @robertbwatson 
Should we be going through the facade to get these values as we do in lines 282 and 296? I'm pretty sure anytime we interact with the settings, we want to use the UTIL_CustomSettingsFacade.getSettings() method

Reply:
@lifewithryan  looks like this is an outdated file. It is already fixed.